pkg/gds/mock.go

```diff
@@ -46,6 +46,11 @@ func NewMock(conf config.Config) (s *Service, err error) {
 	if admin.tokens, err = tokens.MockTokenManager(); err != nil {
 		return nil, err
 	}
+	gin.SetMode(admin.conf.Mode)
+	admin.router = gin.New()
+	if err = admin.setupRoutes(); err != nil {
+		return nil, err
+	}
```

Comment:
If we get too much closer to the `NewAdmin` function, we might want to consider having that setup the mock instead of doing it in this function. Let's keep an eye on it.